Publish/unpublish to active (#304)
mautic/user-documentation: docs/channels/focus_items.rst

**Category** - Assign a Category to help you organize your Focus Items.

**Active** - Whether the Focus Item is available for use - active - or not available - inactive.

.. vale off

**Activate at (date/time)** - This allows you to define the date and time at which this Text Message is available for sending to Contacts

> **Disable at (date/time)** - This allows you to define the date and time at which this Text Message ceases to be available for sending to Contacts.

Reviewer: ```suggestion
**Deactivate at (date/time)** - This allows you to define the date and time at which this Text Message ceases to be available for sending to Contacts.
```

**Category** - Assign a Category to help you organize your Focus Items.

> **Active** - Whether the Focus Item is available for use - active - or not available - unactive

Reviewer: ```suggestion
**Active** - Whether the Focus Item is available for use - active - or not available - inactive.
```

Using Focus Items
*****************

> Once you have created your Focus Item, you're ready to activate it to your website. If you're not quite ready for the Focus Item to go live but you need to get it set up on your website, you can set the Focus Item to unactive.

Reviewer: ```suggestion
Once you have created your Focus Item, you're ready to activate it to your website. If you're not quite ready for the Focus Item to go live but you need to get it set up on your website, set the Focus Item to inactive.
```
